### PR TITLE
MNT: Avoid issues when somehow we try to disconnect from an already…

### PR DESCRIPTION
**Pull request by hhslepicka**
… disconnected channel.

### CODE REVIEW

File: pydm/data_plugins/epics_plugins/pyepics_plugin_component.py [hunk 1]
```diff
@@ -156,19 +156,19 @@ def remove_listener(self, channel):
         if channel.value_signal is not None:
             try:
                 channel.value_signal[str].disconnect(self.put_value)
-            except KeyError:
+            except (KeyError, TypeError):
```
Comment (hmbui) [inline pydm/data_plugins/epics_plugins/pyepics_plugin_component.py]: Why can't we just have a single try - catch for all these disconnections? If one of these disconnections has a problem, can we know which value_signal is causing the issue by looking at the call stack?

File: pydm/data_plugins/plugin.py [hunk 2]
```diff
@@ -91,25 +91,46 @@ def remove_listener(self, channel):
                 pass
 
         if channel.severity_slot is not None:
-            self.new_severity_signal.disconnect(channel.severity_slot)
+            try:
```
Comment (hmbui) [inline pydm/data_plugins/plugin.py]: Could we have a single disconnect() function that takes a signal and a slot? Then, we'll have exception handling in just one place.

File: pydm/data_plugins/epics_plugins/pyepics_plugin_component.py (hunk 1 above)
Comment (hmbui) [reply pydm/data_plugins/epics_plugins/pyepics_plugin_component.py]: Or have a disconnect() function that takes a signal and a slot, and have exception handling in one place (I have the same comment for the changes below).

Comment (hhslepicka) [reply pydm/data_plugins/epics_plugins/pyepics_plugin_component.py]: @hmbui that is a temporary fix to solve the problem described at #177 (which is an ancient issue) and now I have some clues to what it probably is... this is the easiest fix to get the app going. What I will do is to get back to that issue and fix properly by next week.

**Pull request merged**